spec/classes/kibana_spec.rb

```diff
@@ -159,7 +159,7 @@
 
             context 'with bad parameters' do
               {
-                'server.basePath' => '',
+                'server.basePath' => 4.2,
```

Comment:
@bastelfreak Thanks for the feedback.  Before this change, string values in `kibana::config` had to be at least 1 character long.  That test was expected to fail because the string value was 0 characters.  Since this change intentionally allows 0-character strings, I changed the test to ensure that floating-point values cause compilation to fail (since the `Variant[…]` type list does not include the `Float` type).